instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/snsattributes_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otelaws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/sns"
+	"github.com/aws/aws-sdk-go-v2/service/sns/types"
+	"github.com/aws/smithy-go/middleware"
+	"github.com/stretchr/testify/assert"
+
+	semconv "go.opentelemetry.io/otel/semconv/v1.27.0"
+)
+
+func TestPublishInput(t *testing.T) {
+	input := middleware.InitializeInput{
+		Parameters: &sns.PublishInput{
+			TopicArn: aws.String("arn:aws:sns:us-east-2:444455556666:my-topic"),
+		},
+	}
+
+	attributes := SNSAttributeSetter(context.TODO(), input)
```

Comment:
Why `context.TODO` and not `context.Background` here?

Reply:
👋🏽 Happy to change. context.TODO was being used in SQS https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/sqsattributes_test.go#L37